homeassistant/components/evohome/climate.py

```diff
@@ -362,15 +409,19 @@ def current_temperature(self) -> float | None:
         Controllers do not have a current temp, but one is expected by HA.
         """
         temps = [
-            z.temperatureStatus["temperature"]
+            z.temperatureStatus["temperature"]  # type: ignore[index,unused-ignore]
```

Comment:
Note use of `unused-ignore`. The original error was:
`Value of type "dict[str, Any] | None" is not indexable  [index]`

Reply:
`temperatureStatus` can be `None` according to typing, so that has to be taking into account.